<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnTable1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable8">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable9">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTableS1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table S1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTableS2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table S2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTableS3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table S3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTableS4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table S4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTableS5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table S5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTableS6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table S6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTableS7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table S7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTableS8">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table S8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTableS9">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Table S9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrunch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Brunch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCoffee">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Coffee</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFood">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Food</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDrinks">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Drinks</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinQuantity">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddToOrder">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD TO ORDER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrintReceipt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PRINT RECEIPT</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetOrder">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineComm">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPDA_System">
    <property name="title">
     <string>PDA System</string>
    </property>
   </widget>
   <addaction name="menuPDA_System"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
